README.md

```
@@ -64,9 +64,8 @@ To set up Clowder locally, follow these steps:
 5. Run the Development Server
     Start the local development server:
 
+   ```bash
     npm run dev
     # or
     yarn dev
-
-
-Your application should now be running on http://localhost:3000.
\ No newline at end of file
+Your application should now be running on http://localhost:3000.
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)